hasweb/pages/site/adtos.md

```diff
@@ -1,9 +1,17 @@
-title: Sponsorship Terms of Service
+title: Sponsorship and advertising Terms of Service
 
-## You must agree to these terms to advertise with HasGeek
+HasGeek makes physical and online space available for advertisers to showcase their products, APIs and services, for a fee. Specific terms for such opportunities will apply and will be set forth separately (in an email, ordering facility, written agreement, etc). The advertising order will be considered fulfilled once the Advertiser’s collateral has been displayed or serviced for a pre-specified period of time. 
+
+Advertisers may cancel their order through Customer's account if online cancellation functionality is available, or, if not available, with prior written notice to HasGeek, including without limitation electronic mail.  The cancellation of any advertising may be subject to HasGeek’s Advertising Program policies or HasGeek’s ability to re-schedule reserved inventory or cancel ads already in production. Cancelled ads may be published despite cancellation if cancellation of those ads occurs after any applicable commitment date as set forth in advance by HasGeek, in which case Advertiser must pay for those ads.
+
+## You must agree to these terms to advertise with HasGeek:
 
 I understand that my advertisement is subject to full specifications and artwork provided by me at least 72 hours before the scheduled start of such advertisement.
 
 I understand that my interactions with participants is my responsibility, and will not solicit for an illegitimate work opportunities and will not use sexist, racist or misogynist language on advertising collateral. I further assert that I am solely responsible for the content of my listing.
 
 HasGeek may revise these terms of service or discontinue the service without prior notice at any time. If the advertising order is made and involved a payment, a credit note will be issued.
+
+## Grievances
+
+In case you feel that the product or service received is not as per your expectations, you must bring it to the notice of our customer service at <support@hasgeek.com> within 24 hours of receiving the product or service. The Customer Service Team, after looking into your complaint, will take an appropriate decision.
```

Comment:
Reporting grievances is not mandatory, so maybe you can replace 'you must bring it to the notice of our customer service at' with 'please notify us at'?